### DISCUSSION TITLE
Can custom default settings be defined at **Select Files or Folders** screen?

### DISCUSSION THREAD

**Discussion by TFWol** [WinMerge/winmerge#1879] [Q&A] [answered]
I'm trying to figure out if settings like **Include subfolders** at the **Select Files or Folders** screen are able to have a default setting for when you start up the program.

From what I can see, when you change the setting anywhere else in the program, the last setting will be used when you start the program up again. I'd rather it revert to a default.

**Comment by sdottaka** [comment, marked as answer]
To put "Include Subfolders" in a checked state at startup, add `/cfg Recurse=1` to the WinMerge command line argument. You may want to create a WinMerge shortcut on your desktop and add that command line argument to it.

**Reply by TFWol** [reply]
Thanks, it looks like I'll have to do the same with other similar settings.
